qds_sdk/qubole.py

```diff
@@ -21,6 +21,8 @@ class Qubole:
     """
 
     MIN_POLL_INTERVAL = 1
+    RETRIES_CAP = 10
```

Comment:
Try to come up with optimal value of retries and delay considering we want to finish all retries within 5 minutes.